docs/Streams.md

```diff
@@ -186,3 +194,21 @@ MsQuic does not enforce it, and it is legal for an application to provide less b
 
 After the initial receive window is full, flow control will ensure that the peer does not send more data than there is buffer space available.
 However, the application should still provide enough buffer space to keep flow control from impacting performances.
+
+## Receive Shutdown
+
+The receiver can abortively shutdown a stream receive direction by calling [`StreamShutdown`](api/StreamShutdown.md) 
+with the `QUIC_STREAM_SHUTDOWN_FLAG_ABORT_RECEIVE` option.
+
+# Closing a Stream
+
+Once all the directions relevant for a stream have been shutdown, the application receives a
```

Comment:
All the directions relevant? Are there more directions than send and receive? 😄  Joking aside, I get that you're trying to cover unidirectional and bidirectional streams in one statement. Maybe something like 
> Once both directions for a stream have been shutdown (unidirectional streams start with one direction already shutdown) ...

Reply:
Yes, I've been struggling to find a wording that is correct for both uni and bi-directional streams + the send or receive direction.

Maybe "Once a stream has been shutdown (both directions for a bi-directional stream), ..."